### PR TITLE
NAS-105377 / 12.0 / Nas 105377 1

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

File: src/app/views/sessions/signin/signin.component.ts [hunk 1]
```diff
@@ -342,6 +343,15 @@ export class SigninComponent implements OnInit, OnDestroy {
...
   gotoTC() {
-    window.open(this.tc_ip);
+    this.dialogService.generalDialog({
+      title: helptext.tcDialog.title,
+      message: helptext.tcDialog.message,
+      is_html: true,
+      confirmBtnMsg: helptext.tcDialog.confirmBtnMsg,
+    }).subscribe(res => {
+      if (res) {
+        window.open(this.tc_ip);
```
Comment (william-gr) [inline src/app/views/sessions/signin/signin.component.ts:353]: where does TC IP come from? We are probably going to have two attributes, one for URL and one for IP

File: src/app/helptext/topbar.ts [hunk 2]
```diff
@@ -57,6 +57,12 @@ First-time connections require signing up with TrueCommand Cloud.'),
 Login or signup is required.'),
         enabled_placeholder: T('Enable'),
         enabled_tooltip: T('Immediately connect to TrueCommand.'),
+    },
+
+    tcDialog: {
+        title: T('Go to TrueCommand UI'),
```
Comment (Mrt134) [inline src/app/helptext/topbar.ts:63]:
```suggestion
        title: T('Open TrueCommand User Interface'),
```

File: src/app/helptext/topbar.ts [hunk 3]
```diff
@@ -57,6 +57,12 @@ First-time connections require signing up with TrueCommand Cloud.'),
...
+        message: T(`<em>Warnning:</em> TrueCommand UI only connectable if wireguard has been actived.`),
```
Comment (Mrt134) [inline src/app/helptext/topbar.ts:64]:
```suggestion
        message: T(`<em>Warning:</em> The Wireguard service must be active to access the TrueCommand UI.`),
```

Comment (Mrt134) [reply src/app/helptext/topbar.ts:64]: definitely - I'll update the suggestion

**Pull request merged**